Rover/mode_circle.cpp

```diff
@@ -168,10 +175,10 @@ void ModeCircle::update()
     target.vel.rotate(target.yaw_rad + radians(90));
 
     // acceleration is towards center of circle and is speed^2 / radius
-    target.accel = Vector2f(sq(target.speed) / config.radius, 0);
-    target.accel.rotate(target.yaw_rad + radians(180));
+    target.accel = Vector2f(-sq(target.speed) / config.radius, accel_fb);
```

Comment:
I think we need to divide out by time to get back to a acceleration. To cancel out the `* rover.G_Dt` on 158.
```suggestion
    target.accel = Vector2f(-sq(target.speed) / config.radius, accel_fb / rover.G_Dt);
```

Reply:
Thanks very much for finding this.  I've fixed and re-tested and we do see a small improvement as shown below.

![rover-circle-before-vs-after](https://github.com/ArduPilot/ardupilot/assets/1498098/68e83c5a-84ab-460f-8142-c3bc172157ac)